### ISSUE TITLE
Some flux error details being masked as "internal error"

### ISSUE THREAD

**Issue by mark-rushakoff** [influxdata/influxdb#15237]
I was trying to build out a query and I was receiving an unhelpful `internal error` message.

I used this simple query as a reproducer:

```
import "influxdata/influxdb/v1"
buckets() 
  |> v1.fieldsAsCols()
```

I don't expect that query to work correctly, but I do expect a useful error message.

Through the Chronograf script editor, I was getting the HTTP response `{"code":"internal error","message":"An internal error has occurred"}`, which was being displayed in the UI as just `An internal error has occurred`. This error message does not instruct the user on how to resolve the problem. There were no corresponding messages in the influxd log output. 

It appears that this is the same behavior from the CLI with `influx repl`, so it does not seem to be limited to the frontend.

I git bisected to cbd04f28840a000349273e1373180fdf6c79917a (#15196) as the first bad commit.

Prior to that commit, the UI would display a helpful error: `specified column does not exist in table: _time`.

**Comment by jsternberg** [comment]
This specific aspect of pivot is addressed in influxdata/flux#1918.

**Comment by jsternberg** [comment]
This is resolved with v0.50.0 being integrated into influxdb.
